bedrock/firefox/templates/firefox/home.html

```diff
@@ -136,19 +136,14 @@ <h2 class="section-title">{{ _('2x Faster') }}</h2>
           {% endif %}
           </div>
           <div class="key-feature-media video">
-            <div class="moz-video-container">
-              <button class="moz-video-button" type="button" aria-controls="faster-video">{{ _('Watch the video') }}</button>
-              <div class="video-container">
-                <video id="faster-video" preload="none" controls poster="{{ static('img/firefox/home/quantum-vs-chrome-poster.png') }}" data-ga-label="Quantum vs Chrome Video">
-                  <source src="https://assets.mozilla.net/video/firefox-home/quantum-vs-chrome.webm" type="video/webm">
-                  <source src="https://assets.mozilla.net/video/firefox-home/quantum-vs-chrome.ogv" type="video/ogg; codecs='theora, vorbis'">
-                  <source src="https://assets.mozilla.net/video/firefox-home/quantum-vs-chrome.mp4" type="video/mp4">
-                  <a href="https://www.youtube.com/watch?v=YIywpvHewc0">
-                    <img src="{{ static('img/firefox/home/quantum-vs-chrome-poster.png') }}" alt="">
-                  </a>
-                </video>
-              </div>{#-- /.video-container --#}
-            </div>{#-- /.moz-video-container --#}
+            <video preload="none" muted controls playsinline poster="{{ static('img/firefox/home/quantum-vs-chrome-poster.png') }}" data-ga-label="Quantum vs Chrome Video"
```

Comment:
This video has audio. Not sure if adding `muted` here is intentional?

Reply:
🍝